servicex_for_trexfitter/read_trex_config.py

```diff
@@ -0,0 +1,105 @@
+import re
+import linecache
+import json
+
+def read_configuration(confFile: str, block_name: str):
+    """
+    Read TRExFitter configuration file and return block as python dict
+    """
+    Block = {}
```

Comment:
Somewhat unusual to have local variables with capitalized names

Reply:
changed